src/app/header/header.component.ts

```diff
@@ -25,25 +26,9 @@ export class HeaderComponent {
       browser_alert.hidden = false;
     }
 
-       // Set the environment
-       switch (this.apiService.env) {
-        case 'local':
-          this.envName = 'Local';
-          break;
-        case 'dev':
-          this.envName = 'Development';
-          break;
-        case 'test':
-          this.envName = 'Testing';
-          break;
-        case 'demo':
-          this.envName = 'Demo';
-          break;
-        case 'hotfix':
-          this.envName = 'Hotfix';
-          break;
-        default:
-          this.envName = null;
-      }
+    this.envName = this.apiService.env;
+    this.bannerColour = this.apiService.bannerColour;
+    console.log(this.bannerColour);
```

Comment:
Still a console log.

No that this is a big deal, bit now the banner is not going to read the nicely formatted name ie. 'Development.' Instead it is going to just say 'dev.' Probably okay, but initially this was supposed to match MDS, which is the other way.